main.tf

```
@@ -47,4 +47,12 @@ resource "local_sensitive_file" "private_key_pem" {
   content         = tls_private_key.default[0].private_key_pem
   filename        = local.private_key_filename
   file_permission = "0600"
-}
\ No newline at end of file
+}
+
+resource "aws_ssm_parameter" "private_key" {
+  count = local.enabled && var.ssm_parameter_enabled == true ? 1 : 0
+  name  = format("%s/%s", var.ssm_parameter_path_prefix, module.this.id)
```

Sorry, I missed this, but if we include the trailing `/` then the parameter by default will be `/ec2/key-pairs//$module.this.id`, which we don't want. Removing it here should address that. 
```suggestion
  name  = format("%s%s", var.ssm_parameter_path_prefix, module.this.id)
```

---

you're right, thanks for fixing this one 🙌